src/ffi/unicodeobject.rs

```diff
@@ -77,10 +77,9 @@ extern "C" {
     ) -> *mut PyObject;
     #[cfg_attr(PyPy, link_name = "PyPyUnicode_FromObject")]
     pub fn PyUnicode_FromObject(obj: *mut PyObject) -> *mut PyObject;
+    // #[cfg_attr(PyPy, link_name = "PyPyUnicode_FromFormatV")]
+    // pub fn PyUnicode_FromFormatV(format: *const c_char, vargs: va_list) -> *mut PyObject;
```

Comment:
I'm not sure what's up with this..it seems that all functions using the `va_list` are commented out?

Reply:
I guess it's because support in std is not stable? We could enable it behind an optional flag.

https://doc.rust-lang.org/std/ffi/struct.VaList.html

Reply:
Ah, I had no idea that existed. Given that it's been on nightly for 3 years now and it doesn't look like it's close to stabilization, I think I prefer just not having these.